src/test/java/com/softserve/academy/spaced/repetition/controller/AuditControllerTest.java

```diff
@@ -24,13 +25,28 @@
 import java.util.List;
 
 import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
 
 
 @RunWith(MockitoJUnitRunner.class)
 public class AuditControllerTest {
+    @Test
+    public void getAuditByPage() throws Exception {
+        int numberPage = 1;
+        String sortBy = "id";
+        boolean ascending = true;
+        when(auditService.getAuditByPage(numberPage, sortBy, ascending)).thenReturn(createAudit());
+        mockMvc.perform(get("/api/admin/audit?p=1&sortBy=id&asc=true")
+                .accept(MediaType.APPLICATION_JSON)
+                .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.numberOfElements", Matchers.is(10)));
```

Comment:
Could you please add few other asserts like "size of returned collection, pageNumber and other fields of Page object"?

Reply:
Done!